brave/src/main/java/brave/internal/recorder/MutableSpan.java

```diff
@@ -13,11 +14,10 @@
   // Since this is not exposed, this class could be refactored later as needed to act in a pool
   // to reduce GC churn. This would involve calling span.clear and resetting the fields below.
   MutableSpan(TraceContext context, Endpoint localEndpoint) {
-    this.span = zipkin.internal.v2.Span.builder()
-        .traceIdHigh(context.traceIdHigh())
-        .traceId(context.traceId())
-        .parentId(context.parentId())
-        .id(context.spanId())
+    this.span = zipkin.internal.v2.Span.newBuilder()
+        .traceId(HexCodec.toLowerHex(context.traceIdHigh(), context.traceId()))
```

Comment:
this stuff will be replaced with moving the string form to TraceContext, where it will be immutable